.github/ISSUE_TEMPLATE/2_bug_report_macos.yml

```diff
@@ -0,0 +1,71 @@
+name: "Report a macOS Issue"
+description: "Create a report for a runtime related macOS Issue"
+labels: ["os:mac"]
+body:
+- type: textarea
+  attributes:
+      label: "Important Information"
+      description: >
+          - mpv bersion
+           - macOS bersion
+           - Source of the mpv binary or bundle
+           - If known which version of mpv introduced the problem
+           - Possible screenshot or video of visual glitches
+
+          If you're not using git master or the latest release, update.
+          Releases are listed [here](https://github.com/mpv-player/mpv/releases).
+      value: |-
+          - mpv version:
+          - macOS version:
+          - Source:
+          - Introduced in version:
+  validations:
+      required: true
+- type: textarea
+  attributes:
+      label: "Reproduction Steps"
+      description: >
+          Try to reproduce your issue with `--no-config` first. If it isn't reproducible
+          with `--no-config` try to find out which option or script causes your issue.
+
+          If the issue is performance-related, try to reproduce it with `--no-config
+          --profile=fast`, which is designed to work on lower-end mobile devices.
+          Additionally, enable hardware decoding with `--hwdec=yes` or the `Ctrl+h` shortcut.
+
+          Describe the reproduction steps as precise as possible. It's very likely that
+          the bug you experience wasn't reproduced by the developer because the workflow
+          differs from your own.
+  validations:
+      required: true
+- type: textarea
+  attributes:
+      label: "Expected Behavior"
+  validations:
+      required: true
+- type: textarea
+  attributes:
+      label: "Actual Behavior"
+  validations:
+      required: true
+- type: textarea
+  attributes:
+      label: "Log File"
+      description: >
+          Make a log file made with `-v -v` or `--log-file=output.txt`, attach it to
+          the issue. If you use the Bundle, a default log is created for your last run at
+          `~/Library/Logs/mpv.log`. You can jump to that file via the `Help > Show log File…`
+          menu.
+
+          In the case of a crash, please provide the macOS Crash Report (Backtrace).
+
+          Without the log file, this issue will be closed for ignoring the issue template.
+  validations:
+      required: true
+- type: textarea
+  attributes:
+      label: "Sample Files"
+      description: >
+          Sample files needed to reproduce this issue can be attached to the issue
+          (preferred), or be uploaded to https://0x0.st/ or similar sites.
+          (Only needed if the issue cannot be reproduced without it.)
+          Do not use garbage like "cloud storage", especially not Google Drive.
```

Comment:
What's wrong with Google Drive?

Reply:
it's google.

though more seriously aren't we scope creeping this PR a bit? this PR was meant to transition to forms and not to rewrite too much. i am not against such changes, though we might lose focus on the essential question how we want to implement the forms in the end?

Reply:
I don't like specifically mentioning services. For what? `Do not use garbage like "cloud storage", especially not Google Drive.` doesn't bring much value when user reads it. How about we make it less angry and do

> Make sure to using reliable solutions that offer easy to access storage with long-term retention.

Or just remove it all together. I would try to keep template simple without unneeded narrative.

Reply:
reworded it and added a list of some possible services to pick from.

imo not providing any suggestion where to upload samples will just lead to no samples being upload. we have to keep the threshold low. one way is to suggest familiar services.

tbh it would be the best of upload restriction on github weren't so ridiculous dumb sometimes.